java-tests/build.gradle.kts

```diff
@@ -46,12 +56,20 @@ val forMcJava = setOf("extensions", "extra-definitions")
 subprojects {
     if (project.name in forMcJava) {
         apply(plugin = Spine.McJava.pluginId)
+        val validationVersion: String by extra
+        configurations.all {
+            resolutionStrategy {
+                dependencySubstitution {
+                    substitute(module("io.spine.validation:spine-validation-java-bundle")).using(project(":java"))
```

Comment:
I saw this substitution several months ago already :)

Let's have a comment, explaining the idea. Just in case.